kchd/hardware.py

```diff
@@ -10,16 +10,16 @@ class KCHLED(enum.IntEnum):
     The value of each LED in the Enum is the BCM pin number.
     """
 
-    BOOT_20 = 7
-    BOOT_40 = 5
+    # BOOT_20 = 7  # Controlled by hat EEPROM GPIO map
+    # BOOT_40 = 5  # Controlled by systemd
     BOOT_60 = 12
     BOOT_80 = 6
     BOOT_100 = 13
 
     CODE = 11
     COMP = 16
 
-    HEARTBEAT = 19
+    # HEARTBEAT = 19  # Controlled by hat EEPROM device tree
```

Comment:
```suggestion
    HEARTBEAT = 19  # Controlled by hat EEPROM device tree
```

As the `HEARTBEAT` LED is not in use by any of the `LEDController` classes, this LED is never touched by kchd anyway. There is little advantage to commenting it out.

Reply:
> There is little advantage to commenting it out.

Quite the opposite, by removing it from the enum if someone (who may not be aware that it is being driven externally) attempts to use it they will get an error.

Reply:
@raccube Please could we have a third opinion here?